app/controllers/properties_controller.rb

```diff
@@ -37,6 +37,13 @@ def update
   end
 
   def show
+    @resDates = []
```

Comment:
What is this for?

Reply:
This is just part of the code that doesn't work. It was supposed to disable the dates in between once a pair of dates have been booked in order to avoid overlapping booking.